operations/iuf/IUF.md

```diff
@@ -237,19 +243,26 @@ activity:
 [...]
 ```
 
+An input file populated with default values can be created by using `iuf -w`:
+
+```bash
+ncn-m001:~ # iuf -a admin-230127 -m admin-230127/media -i /tmp/default-input-file -w
+Successfully wrote /tmp/default-input-file
+```
```

Comment:
This should follow the command prompt conventions for the CSM docs, as well as the convention to separate the command being run from the example output.

````suggestion
(`ncn-m#`) An input file populated with default values can be created by using `iuf -w`:

```bash
iuf -a admin-230127 -m admin-230127/media -i /tmp/default-input-file -w
```

Example output:

```text
Successfully wrote /tmp/default-input-file
```
````